tools/topology/topology2/include/common/sink_audio_format.conf

```diff
@@ -0,0 +1,137 @@
+# Class definition for sink pin audio format object
```

Comment:
I'm confused.
Don't we have support for pins other than index 0? I would thin that they somehow work and somehow they actually get audio format, no?

Isn't the problem lies that we have only one set of audio format definition, which is global for the whole module and for all of it's pins?

Reply:
@ujfalusi we dont. The only audio formats we use today are for pin 0 and always intput and output together in pairs. Clearly this wont work